lib/taskana-core/src/test/java/pro/taskana/impl/TaskServiceImplTest.java

```diff
@@ -1333,9 +1336,44 @@ public void testUpdateTaskRemovingAttachment() throws TaskNotFoundException, Sys
         assertThat(actualTask.getAttachments().size(), equalTo(0));
     }
 
+    @Test
+    public void testTaskSummaryEqualsHashCode() throws InterruptedException  {
+        Classification classification = createDummyClassification();
+        Workbasket wb = createWorkbasket("WB-ID", "WB-Key");
+        Attachment attachment = JunitHelper.createDefaultAttachment();
+        ObjectReference objectReference = JunitHelper.createDefaultObjRef();
+        TaskImpl taskBefore = createUnitTestTask("ID", "taskName", wb.getKey(), classification);
+        taskBefore.setPrimaryObjRef(objectReference);
+        Thread.sleep(10);
```

Comment:
Why? is there any job working?

Reply:
In order to guarantee that the 'created' timestamps are different. That's the only data that distinguishes taskBefore and taskAfter ...